.azure/gpu-tests-lite.yml

```diff
@@ -0,0 +1,105 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
+  paths:
+    include:
+      - ".azure/gpu-tests-lite.yml"
+      - "requirements/lite/**"
+      - "src/lightning_lite/**"
+      - "tests/tests_lite/**"
+
+pr:
+  - "master"
+  - "release/*"
+
+jobs:
+  - job: testing
+    strategy:
+      matrix:
+        'PyTorch - stable':
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1"
```

Comment:
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/6035284/187959390-2c734708-d6dd-4f9d-abab-77fa02b0f8c2.png)

Reply:
Great. Isn't it a problem if we don't use the` pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1` image?